main.go

```diff
@@ -30,26 +34,31 @@ const (
 func applyVirtualMachineExport(vmName string) error {
 	log.Println("Applying VirtualMachineExport object...")
 
-	yaml := fmt.Sprintf(`
-apiVersion: export.kubevirt.io/v1alpha1
-kind: VirtualMachineExport
-metadata:
-  name: %s
-spec:
-  source:
-    apiGroup: "kubevirt.io"
-    kind: VirtualMachine
-    name: %s
-`, vmName, vmName)
-	cmd := exec.Command("kubectl", "apply", "-f", "-")
-	cmd.Stdin = strings.NewReader(yaml)
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
-
-	if err := cmd.Run(); err != nil {
+	client, err := kubecli.GetKubevirtClient()
+	if err != nil {
 		return err
 	}
-	return nil
+
+	vmNamespace := os.Getenv("VM_NAMESPACE")
```

Comment:
please update task yaml manifest accordingly

Reply:
Wouldn't be the function more easy to test, if the vmNamespace would be a parameter of  applyVirtualMachineExport?

Reply:
> please update task yaml manifest accordingly

I thought it's not needed, and would be properly resolved in a new PR.

> Wouldn't be the function more easy to test, if the vmNamespace would be a parameter of applyVirtualMachineExport?

Yes, this is the plan to do it in a new PR. I just want to get rid of the kubectl package and not change the API.

Reply:
I'll resolve this issue in a new PR https://github.com/codingben/kubevirt-disk-uploader/pull/49.